Content.Shared/Follower/FollowerSystem.cs

```diff
@@ -249,20 +252,33 @@ public void StopFollowingEntity(EntityUid uid, EntityUid target, FollowedCompone
     }
 
     /// <summary>
-    /// Get the most followed entity.
+    /// Gets the entity with the most non-admin ghosts following it.
     /// </summary>
-    public EntityUid? GetMostFollowed()
+    public EntityUid? GetMostGhostFollowed()
     {
         EntityUid? picked = null;
-        int most = 0;
-        var query = EntityQueryEnumerator<FollowedComponent>();
-        while (query.MoveNext(out var uid, out var comp))
+        var most = 0;
+
+        // Keep a tally of how many ghosts are following each entity
+        var followedEnts = new Dictionary<EntityUid, int>();
+
+        // Look for followers that are ghosts and are player controlled
+        var query = EntityQueryEnumerator<FollowerComponent, GhostComponent, ActorComponent>();
+        while (query.MoveNext(out var uid, out var follower, out var _, out var _))
         {
-            var count = comp.Following.Count;
-            if (count > most)
+            // Exclude admins
+            if (_adminManager.IsAdmin(uid))
+                continue;
+
+            var followed = follower.Following;
+            // Add new entry or increment existing
+            followedEnts.TryGetValue(followed, out var currentValue);
+            followedEnts[followed] = currentValue + 1;
+
+            if (followedEnts[followed] > most)
             {
                 picked = uid;
```

Comment:
Forgot to switch this to the followed entity.
```suggestion
                picked = followed;
```

Reply:
Oops, thank you